aip_xx1_gen2_launch/launch/radar.launch.xml

```diff
@@ -25,7 +25,7 @@
   </group>
 
   <group if="$(eval &quot;'$(var ars_version)'=='ars548'&quot;)">
-    <let name="odometry_topic" value="/sensing/vehicle_velocity_converter/twist_with_covariance"/>
+    <let name="odometry_topic" value="/localization/kinematic_state"/>
```

Comment:
```suggestion
```
I haven't checked the all configs but as I remember each default `odometry_topic` is `/localization/kinematic_state` ...? Any exception?

Reply:
No, this radar.launxh.xml calls [continental_launch_all_hw.xml in Nebula](https://github.com/tier4/nebula/blob/main/nebula_ros/launch/continental_launch_all_hw.xml#L13), but the odometry_topic is set to a meaningless topic by default in that nebula file. 

Therefore, we need to specify the correct odometry_topic here.

Reply:
Thank you, make sense!